### PR TITLE
[ODM-9057] update genestack user setup doc

### PR DESCRIPTION
**Pull request by dgm90**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,6 @@
+* 0.35.6
+    * Update documentation regarding user setup
+
```
Comment (klas-genestack) [inline ChangeLog:3]: I don't think we should release a new version at all.  The changes are 100% in user documentation, so updating package on user's side makes absolutely zero difference.

**Comment by genestack-okunitsyn** [comment]
lgtm!

File: ChangeLog (hunk 1 above)
Comment (klas-genestack) [reply ChangeLog:3]: I propose to make doc change with no Python Client release.  We have already released four versions with absolutely zero changes in behaviour.  I am prepared to rebuild the docs manually myself to skip this whole release hassle, so please ditch these version bumps if possible.

Comment (genestack-okunitsyn) [reply ChangeLog:3]: Ofc you can update documentation manually. As you want. :)

Sorry, but ditch version bumps not possible, it's a preparation for future release process update.

Comment (dgm90) [reply ChangeLog]: Discussed in DM with Oleg: I removed the version bump and Klas can proceed with the manual doc release

**Review by klas-genestack — APPROVED** [review]

**Comment by klas-genestack** [comment]
[Done](https://genestack-client.readthedocs.io/en/stable/getting_started.html#configuring-credentials)

**Pull request merged**